r/tools/nixlibs.R

```diff
@@ -19,7 +19,7 @@ args <- commandArgs(TRUE)
 VERSION <- args[1]
 dst_dir <- paste0("libarrow/arrow-", VERSION)
 
-arrow_repo <- "https://arrow-r-nightly.s3.amazonaws.com/libarrow/"
+arrow_repo <- "https://nightly.wujciak.de/r/libarrow/"
```

Comment:
revert before merge

Reply:
These should probably also use the `options(arrrow.dev_repo)` like in install_arrow

Reply:
https://github.com/apache/arrow/pull/13173